frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -94,9 +91,7 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({
         )}
       </dd>
       <dt>IP Address</dt>
-      <dd id={prefixedID(id, 'ip-addresses')}>
-        {ipAddresses.length > 0 ? ipAddresses.join(', ') : DASH}
-      </dd>
+      <dd id={prefixedID(id, 'ip-addresses')}>{getVmiIpAddressesString(vmi, vmStatus) || DASH}</dd>
```

Comment:
In a follow-up, I will update this page to render `Not available` instead of the dash.